src/styles/main.scss

```
@@ -1,3 +1,203 @@
-body {
+* {
+  box-sizing: border-box;
   margin: 0;
+  padding: 0;
+}
+
+html,
+body {
+  font-family: "Roboto", Helvetica, sans-serif;
+}
```

```suggestion
html {
  font-family: Roboto, sans-serif;
}
```
enough to set for html.
looks like somethin went wrong with font. try to set font-family such as in suggestion

---

Still not fixed from the previous review